.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -23,6 +23,11 @@ TODO
 
 ### 🧑‍💻  Steps to reproduce
 <!-- Provide step-by-step actions of how to recreate this bug in a clean install of Fleet. (This helps others understand and fix it more quickly.) -->
+
+These steps:
+[] Have been confirmed to consistently lead to reproduction in multiple Fleet instances.
+[] Describe the workflow that led to the error, but are not consistently repeatable in multiple Fleet instances. 
```

Comment:
Maybe this line could be "Describe the workflow that led to the error, but have not yet been reproduced in multiple Fleet instances" if we want to stick to this format. The existing wording implies to me that we tried to repro the bug but were unable to.

Reply:
Works for me!